server/api/products.js

```diff
@@ -0,0 +1,49 @@
+const router = require("express").Router();
+const {
+  models: { Product },
+} = require("../db");
+
+// /api/products
+
+router.delete("/:id", async (req, res, next) => {
+  try {
+    const product = await Product.findByPk(req.params.id);
+    await product.destroy();
+    res.sendStatus(204);
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.put("/:id", async (req, res, next) => {
```

Comment:
do you want the ability for just anyone to be able to modify your store? (none of these routes are protected)